trimmer_decorator.rb

```diff
@@ -2,8 +2,6 @@
 class TrimmerDecorator < CorrectNameable
   def correct_name
     name = @person.correct_name.strip
-    return name[0..10] unless name.length < 10
-
-    name
+    name[0..9]
```

Comment:
- [ ] [OPTIONAL] It would be nice if you implement your `def` like that:

```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```